### PR TITLE
Add author and maintainer

### PR DESCRIPTION
**Pull request by Levi-Armstrong**
@gavanderhoorn This work to solve issue #1 ?

### CODE REVIEW

File: noetic/package.xml [hunk 1]
```diff
@@ -3,8 +3,8 @@
   <name>taskflow</name>
   <version>:{version}</version>
   <description>The taskflow package</description>
-
-  <maintainer email="user@todo.todo">user</maintainer>
+  <author>Dr.Tsung-Wei Huang</author>
+  <maintainer email="twh760812@gmail.com">Dr.Tsung-Wei Huang</maintainer>
```
Comment (gavanderhoorn) [inline noetic/package.xml]: I would suggest to list you as a maintainer here, as you're the maintainer of the ROS release (I assume?).

The `author` is then the author of the actual source code.

I would further suggest to add a link to their website (`https://github.com/taskflow/taskflow` and / or `https://taskflow.github.io`).

Something like:

```suggestion
  <maintainer email="levi.armstrong@swri.org">Levi Armstrong</maintainer>
  <url type="website">https://taskflow.github.io</url>
  <url type="repository">https://github.com/taskflow/taskflow</url>
```

**Comment by Levi-Armstrong** [comment]
@gavanderhoorn Thank you for the support.

**Comment by Levi-Armstrong** [comment]
@gavanderhoorn Should I wait for the rosdep PR to be merged before running bloom release again or is it OK to run through it again now?

**Comment by gavanderhoorn** [comment]
hm, not sure.

Bloom will fetch the release repository, so it should be able to figure out you're releasing a package again, even if your `ros/rosdistro` PR hasn't been merged yet.

But I haven't tried this myself, so I'm just speculating.

You could just go through the process and see what Bloom does. You might have to provide it with the URL to your release repository again though (as it can't retrieve it yet from the `distribution.yaml`).

If it doesn't appear to work, you could just abort. As long as nothing gets pushed to your release repository that should be fine.

**Pull request merged**